pytorch_lightning/trainer/training_loop.py

```diff
@@ -288,7 +288,9 @@ def train(self):
         model = self.get_model()
 
         # load data
-        self.reset_train_dataloader(model)
+        # if reload_dataloaders_every_epoch, this is moved to the epoch loop
+        if not self.reload_dataloaders_every_epoch:
+            self.reset_train_dataloader(model)
         self.reset_val_dataloader(model)
```

Comment:
this should be aslo in the iff block, right?

Reply:
Do you mean reset_val_dataloader ? If so, in the original code it wasn't reloaded every epoch so I kept the same behaviour.